source/puppet/4.2/reference/upgrade_major_server.markdown

```diff
@@ -75,41 +75,14 @@ Puppet 4 uses different HTTPS URLs to fetch configurations. Any rules in `auth.c
 To convert the URLs:
 
 1. Open your old [`auth.conf`][auth.conf] file, which is probably at `/etc/puppet/auth.conf`.
-2. Identify [any _custom_ rules](/puppetserver/2.1/compatibility_with_puppet_agent.html#transfer-and-modify-custom-authconf-rules) you've added to your old [`auth.conf`][auth.conf] file. (Don't worry about default rules.)
+2. Identify [any _custom_ rules](/puppetserver/latest/compatibility_with_puppet_agent.html#transfer-and-modify-custom-authconf-rules) you've added to your old [`auth.conf`][auth.conf] file. (Don't worry about default rules.)
 3. Change the `path` of each custom rule to use Puppet 4 URLs.
     1. Add `/puppet/v3` to the beginning of most paths.
-    2. The `certificate_status` endpoint ignores `auth.conf`. Configure access in Puppet Server's [`ca.conf`][ca.conf] file.
-4. Add the custom rules to Puppet Server's new `/etc/puppetlabs/puppet/auth.conf` file.
+    2. Configure the `certificate_status` endpoint in `auth.conf`. Puppet Server's `ca.conf` file is [deprecated][] as of Puppet Server 2.2.
+4. (Optional) Convert your rules to use new authorization methods and `auth.conf` format. See the [Puppet Server configuration documentation][/puppetserver/latest/configuration.html] for details.
+5. Add the custom rules to Puppet Server's new `/etc/puppetlabs/puppet/conf.d/auth.conf` file.
 
-#### `auth.conf` Rule Example
-
-Old Puppet 3 rules:
-
-~~~
-# Puppet 3 auth.conf on the master
-path ~ ^/catalog/([^/]+).uuid$
-method find
-allow /^$1\.uuid.*/
-
-# Default rule, should follow the more specific rules
-path ~ ^/catalog/([^/]+)$
-method find
-allow $1
-~~~
-
-New Puppet Server 2 rules supporting both Puppet 3 and 4 agents:
-
-~~~
-# Puppet 3 & 4 compatible auth.conf with Puppet Server 2.1
-path ~ ^/puppet/v3/catalog/([^/]+).uuid$
-method find
-allow /^$1\.uuid.*/
-
-# Default rule, should follow the more specific rules
-path ~ ^/puppet/v3/catalog/([^/]+)$
-method find
-allow $1
-~~~
+For examples of reconciled Puppet Server `auth.conf` rules, in both the legacy Ruby and new HOCON formats, see [the Puppet Server compatibility documentation](/puppetserver/latest/compatibility_with_puppet_agent.html#exampleauthconfrulesforpuppet3and4agents).
```

Comment:
Per @nfagerlund, copy the examples over from the Server docs rather than link to them.